js/widgets/page.js

```diff
@@ -46,6 +46,7 @@ $.widget( "mobile.page", {
 
 		// Deprecated in 1.4 remove in 1.5
 		contentTheme: null,
+		enhanceWithin: true,
```

Comment:
Worse, **r** and **d** are right below one another on the keyboard. I sense a disturbance in the Force.